oracle/config/samples/v1alpha1_cronanything.yaml

```diff
@@ -7,7 +7,6 @@ metadata:
 spec:
   schedule: "*/5 * * * *"
   totalResourceLimit: 10
-  historyCountLimit: 2
```

Comment:
Good catch! It seems historyCountLimit is a subspec under retention. :)

Reply:
Thanks for approval Yian!